Tasks/Npm/npmtoolrunner.ts

```diff
@@ -74,4 +86,47 @@ export class NpmToolRunner extends tr.ToolRunner {
         let config = tl.execSync('npm', `config list ${this.dbg ? '-l' : ''}`, options);
         return options;
     }
+
+    private _getDebugLogPath(options?: tr.IExecSyncOptions): string {
+        // check cache
+        options.silent = true;
+        const cacheLocation = tl.execSync('npm', 'config get cache', options).stdout.trim();
```

Comment:
there's no need to use execSync since we have await now.  It's not terrible if it's a very fast command which doesn't need to stream output or where buffering is not an issue.  We should prefer await tl.exec